sycl/doc/design/spirv-extensions/SPV_INTEL_joint_matrix.asciidoc

```diff
@@ -354,8 +398,12 @@ All operands and the 'Result Type' must be
 <<OpTypeJointMatrixINTEL,*OpTypeJointMatrixINTEL*>> with 'Row Count' equals to
 'M' and 'Column Count' equals to 'N' +
  +
-'Scope' is syncronization scope for operation on the matrix. It must be the
-result of a constant instruction with scalar 'integer type'. +
+'Scope' is syncronization scope for operation on the matrix.
+Must be an '<id>' of a 32-bit integer scalar. Its value is expected to be one
+either _Workgroup_ or _Subgroup_ from the table 3.27. Scope <id>. If validation
+rules or the client API require a constant '<id>', it is invalid for it to not
+be one of these values. If non-constant '<id>' are allowed, behavior is
+undefined if '<id>' is not one of these values. +
```

Comment:
I'm guessing there are some other requirements for the "Use" of the A, B, and C matrix operands that should be documented here?  As an example it is probably an error (or, if it cannot be statically checked, behavior is undefined) if the "Use" for the A matrix is **MatrixB** or **Accumulator**?

Same goes for all other variants of **OpJointMatrixXXMadINTEL**.

Reply:
Applied